content/millennium/dstu2/scheduling/slot.md

```diff
@@ -65,24 +65,26 @@ _Implementation Notes_
  Name             | Required?                                        | Type          | Description
 ------------------|--------------------------------------------------|---------------|------------------------------------------------------------------------------------
 `_id`             | Yes or `slot-type`                               | [`token`]     | The logical resource id associated with the resource.
-`slot-type`       | Yes or `_id`                                     | [`token`]     | The type of appointments that can be booked into this slot. Example: `http://snomed.info/sct|394581000`
-`schedule.actor`  | Yes (or `-location`) when using `slot-type`      | [`reference`] | A single or comma separated list of Practitioner references. Example: `Practitioner/2578010`
-`-location`       | Yes (or `schedule.actor`) when using `slot-type` | [`reference`] | A single or comma separated list of Location references. Example: `633867`
+`slot-type`       | Yes or `_id`                                     | [`token`]     | A single or comma seperated list of appointment types that can be booked into the slot. Example: `http://snomed.info/sct|394581000`
+`schedule.actor`  | No                                               | [`reference`] | The Practitioner references. Example: `Practitioner/2578010`
+`-location`       | No                                               | [`reference`] | The list of Location references. Example: `633867`
 `start`           | Yes when using `slot-type`                       | [`date`]      | The Slot date-time. Example: `2016`
 [`_count`]        | No                                               | [`number`]    | The maximum number of results to be returned per page.
 
-Notes:   
+Notes:
 
-- The `slot-type`, `schedule.actor`, and `-location` parameters may be included only once.
+- The `slot-type` should be provided and may include multiple types separated by a comma
```

Comment:
"The `slot-type` **parameter**"

it doesn't have to be provided when _id is provided, so we'll need to reword "should be provided"

add a period at the end